docker/app/local.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM bitnami/python:3.7-prod
+
+STOPSIGNAL SIGQUIT
+ARG EXTRAS=deploy
+
+# Create a user.
+RUN useradd --system --shell /bin/false --uid 1500 pysite
+
+# Install prerequisites needed to complete the dependency installation.
+RUN install_packages git uwsgi
+
+# Copy the project files into the working directory.
+WORKDIR /app
+COPY . .
+
+# Update setuptools by removing egg first, add other dependencies
+RUN rm -r /opt/bitnami/python/lib/python3.*/site-packages/setuptools* && \
+    pip install --no-cache-dir -U setuptools pipenv
+RUN pipenv install --system --deploy
+
+RUN SECRET_KEY=placeholder DATABASE_URL=sqlite:// python3 manage.py collectstatic --no-input --clear --verbosity 0
+
+CMD ["uwsgi", "--ini", "docker/app/uwsgi.ini"]
```

Comment:
```suggestion
ENTRYPOINT ["uwsgi"]
CMD ["--ini", "docker/app/uwsgi.ini"]
```

Reply:
I'm curious, what's the difference?

Reply:
It lets you override the arguments when you use `docker run` (or in compose I suppose) without having to also write `uwsgi`. I suppose in this case not as useful, but generally you can set the executable + stable args in `ENTRYPOINT` and then put the args that may change in `CMD`.

Reply:
If this was done, the entrypoint still wouldn't be "forced". If you really wanted to change it you could use `docker run --entrypoint`

Reply:
It doesn't work with the docker-compose `command` option, existing with a code 1. Since there's no significant benefit and it's a difference in behaviour, I'll revert to original.

Reply:
That's cause it tries to do `uwsgi migate_and_serve.sh`. In the compose you'd need to make that shell script the entrypoint instead of a command.

Reply:
Exactly, which is an unnecessary change in behaviour with no significant benefit.